.github/workflows/node.js.yml

```diff
@@ -5,26 +5,25 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ dev ]
+    branches: [dev]
   pull_request:
-    branches: [ dev ]
+    branches: [dev]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x]
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: yarn install
-    - run: yarn run fix
-    - run: "[[ -z $(git --no-pager diff -- . ':(exclude)yarn.lock' ':(exclude)yarn.lock') ]]"
-    - run: yarn run test --passWithNoTests
+      - uses: actions/checkout@v2
```

Comment:
i dont think this is due to prettier, this is probably due to your vscode. otherwise i would ask, can you explain why prettier has only started to change it today?

Reply:
I think it's because when I got an error from Node.js saying that I need to add --detectOpenHandles to scripts, I added it to line 29 thinking that it's the place I am supposed to add. Then, I deleted it and saved and prettier formatted the whole document

Reply:
even now when i control Z it to the initial state and save, prettier formats it